File: PRAKTIKUM 2/BAHASA C/PRAK204_2310817220028_SheilaSabina.c
#include <stdio.h>
#include <math.h>

#define phi 3.142857

 void main(){
   double A, B;

   printf("Input\n");
   scanf("%lf %lf", &A, &B);

   double volume = phi * pow(A, 2) * B;
   double luas = 2 * phi * A * (A + B);
   double keliling = 2 * phi * A;

   printf("\nOutput\n");
   printf("Volume = %.2lf\n", volume);
   printf("Luas = %.2lf\n", luas);
   printf("Keliling = %.2lf", keliling);
 }

File: PRAKTIKUM 2/BAHASA PY/PRAK204_2310817220028_SheilaSabina.py
print("Input")
variabel = input().split()

jari_jari, tinggi = map(int, variabel)

phi = 3.142857
volume = phi * (jari_jari**2) * tinggi
luas = 2 * phi * jari_jari * (jari_jari + tinggi)
keliling = 2 * phi * jari_jari

print("\nOutput")
print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")